hack/update-vendor.sh

```diff
@@ -16,6 +16,9 @@
 #limitations under the License.
 ###
 
+KUBEEDGE_ROOT=$(dirname "${BASH_SOURCE[0]}")/..
+
+source "${KUBEEDGE_ROOT}/hack/lib/init.sh"
 
 function checkModified {
 	modified=$( git status --short 2>/dev/null | grep -e "^.M" | wc -l)
```

Comment:
we can delete the `grep -e "^.M"` here. If we only delete files, it's `D` here.

Reply:
ok, use  :  grep -E "^.M|^.D"